### ISSUE TITLE
Character outside editable range isn't detected as error

### ISSUE THREAD

**Issue by rooksdo** [TypeCobolTeam/TypeCobol#1594]
**Description**
When a character is outside the editable text range (further than position 72) the TypeCobol editor doesn't show any error message although compilation is in error.

**Example**
![image](https://user-images.githubusercontent.com/57907033/70737693-68ea6f80-1d13-11ea-8445-029508a2565f.png)

**Expected behavior**
When any character is written after position 72 an error message should appear in the TypeCobol editor.

**Comment by smedilol** [comment]
I don't think that we must detect character after column 72.
The following code compile fines:
![image](https://user-images.githubusercontent.com/13764482/71257959-4ec51880-2335-11ea-99f7-240635dc8b87.png)

We should rather detect incomplete string.
This will allow us to detect other cases like:
![image](https://user-images.githubusercontent.com/13764482/71257996-68fef680-2335-11ea-82ce-7ae3067456fa.png)

**Comment by smedilol** [comment]
Here's a complete example of correct and incorrect continuation line:
```cobol
       IDENTIFICATION division.
       PROGRAM-ID. MyPGM.
       procedure division.
      *         OK
                display 'aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
      -          ' er zer '
      *         KO
      *IGYPS0006-E The first character of a continued alphanumeric
      *literal was not a quote.  A quote was assumed.
                display "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
      -          ' er zer '
      
      *         KO
      *IGYPS0007-E An alphanumeric literal was not properly continued.
      *The end of the literal was assumed on the last source line.
                display "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
      -          " er zer "
      
      *         KO
      *IGYPS0006-E The first character of a continued alphanumeric
      *literal was not a quote.  A quote was assumed.
                display 'aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
      -          " er zer "
      
      *         KO
      *IGYPS0007-E An alphanumeric literal was not properly continued.
      *The end of the literal was assumed on the last source line.
                display 'aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
      *IGYPS0006-E The first character of a continued alphanumeric
      *literal was not a quote.  A quote was assumed.
      -           er zer '
      
                display "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
      *         KO
      *IGYPS0006-E The first character of a continued alphanumeric
      *literal was not a quote.  A quote was assumed.
      -           er zer '
      
      
      *         KO
      *IGYPS0007-E An alphanumeric literal was not properly continued.
      *The end of the literal was assumed on the last source line.
                display "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
      *         KO
      *IGYPS0006-E The first character of a continued alphanumeric
      *literal was not a quote.  A quote was assumed.
      -           er zer "
      
                display 'aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
      *         KO
      *IGYPS0006-E The first character of a continued alphanumeric
      *literal was not a quote.  A quote was assumed.
      -           er zer "
      *         KO
      *IGYPS0007-E An alphanumeric literal was not properly continued.
      *The end of the literal was assumed on the last source line.
           .
      
      
       END PROGRAM MyPGM.
```

**Comment by smedilol** [comment]
@LANOYDO can you check this example of code and update the PR accordingly ?

**Comment by smedilol** [comment]
@LANOYDO Ok my mistake, my examples were all in the same file. So syntax errors at the beginning of the source were conflicting with following line of code.

I've updated my previous example.
Each Program must be put into an independent file, just to be sure that there is no conflict.

**Comment by smedilol** [comment]
Just to be sure I reference the test case from https://github.com/TypeCobolTeam/TypeCobol/issues/1248#issuecomment-573025424

**Comment by rooksdo** [comment]
Fixed by #1604
